src/main/java/dev/meyi/bn/modules/Module.java

```diff
@@ -12,14 +13,14 @@ public abstract class Module {
 
   protected int x;
   protected int y;
-  protected float scale;
+  public float scale;
   int lastMouseX, lastMouseY;
   int boundsX, boundsY;
   int padding = 3;
   int shift = 0;
   int mouseWheelShift = 0;
   boolean moving = false;
-  boolean active = true;
+  public boolean active = true;

```

Comment:
This isn't as important, but if you want, make a getter & setter for it as well. It can be in the module class (same as scale)